packages/jest-cli/src/__tests__/watch.test.js

```diff
@@ -95,6 +97,7 @@ describe('Watch mode flows', () => {
       pipe,
       new TestWatcher({isWatchMode: true}),
```

Comment:
Should we consider changing the signature of run_jest to an object of all this options instead of positional arguments?